dvc/repo/reproduce.py

```diff
@@ -21,7 +22,9 @@ def _reproduce_stage(stage, **kwargs):
         return []
 
     if not kwargs.get("dry", False):
-        stage.dump()
+        print("stage: ", stage.path)
```

Comment:
Why is this print needed?

Reply:
Ooops, was debugging a failed test :smile: 
```suggestion
```